__tests__/index.spec.js

```diff
@@ -38,7 +37,7 @@ describe('json-parse-context', () => {
     Object.entries({
       'string input': (str) => str,
       'Buffer input': (str) => Buffer.from(str),
-    }).forEach(([name, inputTransform]) => describe(name, () => {
+    }).forEach(([name, inputTransform]) => describe(`${name}`, () => {
```

Comment:
curious, what was the problem here?

Reply:
Using a variable for a describe is not allowed